docs/workflow/testing/mono/testing.md

```diff
@@ -1,54 +1,83 @@
 # Running Tests using Mono Runtime
 
+Before running any tests, build mono in the way that you would like to test with.
+See the instructions for [Building Mono](../../building/mono/README.md)
+
 ## Running Runtime Tests
-We currently only support running tests against coreclr.  There are additional mono runtime tests in mono/mono, but they
-have not been moved over yet. Simply run the following command:
+* Build and run runtime tests against Mono JIT:
 
+    * Build and run one specific runtime test. From the root directory run the following command:
```

Comment:
Rather than specifying "from the root directory" each time and then including the cd in both commands, I'd just say "You should run these commands from src/mono/netcore" earlier and then not mention it otherwise. You also don't need the bullet point here, so you can just say "Running a single test:" followed by the code snippet.

Reply:
Fixed